ui/app/document-upload/controllers/documentController.js

```diff
@@ -142,9 +143,21 @@ angular.module('opd.documentupload')
                         v: "custom:(uuid,setMembers:(uuid,name:(name)))"
                     }
                 }).then(function (response) {
+                    if (response.data.results[0].setMembers && response.data.results[0].setMembers.length > 0) {
+                        response.data.results[0].setMembers.forEach(function (concept) {
+                            var conceptToAdd = {
+                                'concept': {
+                                    uuid: concept.uuid,
+                                    name: concept.name.name,
+                                    editableName: concept.name.name
```

Comment:
why editableName on fileType Concept? can we remove this?
below suggested approach for fileType selection, editableName will not be used.